torchsde/_core/methods/srk.py

```diff
@@ -32,11 +32,14 @@
 
 
 class SRK(base_solver.BaseSDESolver):
+    # TODO: should the strong order be 2.0 for additive noise? Numerically it looks like it.
     strong_order = 1.5
     weak_order = 1.5
     sde_type = SDE_TYPES.ito
     noise_types = (NOISE_TYPES.additive, NOISE_TYPES.diagonal, NOISE_TYPES.scalar)
-    levy_area_approximation = LEVY_AREA_APPROXIMATIONS.spacetime
+    levy_area_approximations = [LEVY_AREA_APPROXIMATIONS.spacetime,
```

Comment:
Could we rename `spacetime` to `space_time`? It matches better with the underlying str.

Reply:
Agreed